doc/develop/getting_started/index.rst

```diff
@@ -478,6 +478,10 @@ additional Python dependencies.
 
 .. rst-class:: numbered-step
 
+.. note::
+
```

Comment:
It doesn’t seem necessary to clutter getting started with information about building docs.